spec/item_spec.rb

```diff
@@ -2,43 +2,77 @@
 require_relative "../lib/Standupguy"
 
 describe Standupguy::Item do
+  
+  before { @datafile = create_test_file! }
+  
   describe "#add_to_today" do
-    subject(:item) { Standupguy::Item.new }
 
-    context "for zendesk tickets" do
+    describe "for zendesk tickets" do
       before do
+        @item = Standupguy::Item.new
         expect_zendesk("foo", "api/v2/tickets/1").
           to_return(headers: {"Content-Type" => "application/json"}, body: JSON.dump("ticket" => {"subject" => "test ticket"}))
-        item.add_to_today("https://foo.zendesk.com/tickets/1")
+        @item.add_to_today("https://foo.zendesk.com/tickets/1")
       end
 
       it "includes the ticket subject in the description" do
-        expect(item.data[:description]).to match(/test ticket/)
+        expect(@item.data[:description]).to match(/test ticket/)
       end
 
       it "includes the original url" do
-        expect(item.data[:description]).to match(/foo.zendesk.com\/tickets\/1/)
+        expect(@item.data[:description]).to match(/foo.zendesk.com\/tickets\/1/)
+      end
+
+      it "includes the ticket_id" do
+        expect(@item.data[:ticket_id]).to eq("https://foo.zendesk.com/tickets/1")
+      end
+
+      describe "adding the same ticket again" do
+
+        subject{ Standupguy::Item.new }
+
+        before do
+          @item.save
+          expect_zendesk("foo", "api/v2/tickets/1").
+            to_return(headers: {"Content-Type" => "application/json"}, body: JSON.dump("ticket" => {"subject" => "test ticket"}))
```

Comment:
Line is too long. [129/80]<br>Space inside { missing.<br>Space inside } missing.